### ISSUE TITLE
interface "a la JuMP" with macros

### ISSUE THREAD

**Issue by trigaut** [JuliaStochOpt/StochDynamicProgramming.jl#48]
We figured out with Frapac that it might be clever to create an interface similar to the JuMP one using macros:

@addStateVariable
@addControl
@addDynamics
@addConstraints....

what do you think?

**Comment by leclere** [comment]
Yes that would be nice, it goes along the idea of having an interface that allow easy construction of the problem.
